cookbook/default-snippets.rst

```diff
@@ -0,0 +1,32 @@
+How to define a default-snippet?
+================================
+
+Sulu gives the content-manager an easy way to define default snippets per type.
+This default-snippets automatically be used if the snippet-selection is empty on
+a specific page. Be aware of that this "default" selection will not be displayed
```

Comment:
`~~Be aware of that~~ This default selection will...` Don't make sentences more complicates than they need to be.